### PR TITLE
Feature/better beer caroussel

### PR DESCRIPTION
**Pull request by Bazooo**
- Add x and y for scale and shift
- ParseFloat the values given from the django (fixes the "," given from translating floats to french)
- Modified the svg for the canisters so they have the same size
- Added drag

demo of drag: https://puu.sh/Dk0I6/fd71ad03ec.gif

### CODE REVIEW

File: static/script/beer_carousel.js [hunk 1]
```diff
@@ -34,8 +34,8 @@ function initBeers() {
     $('.beercarousel__controls button.beercarousel__button').eq(0).addClass('beercarousel__button--active');
 }
 
-function tagTransform(scale, pos) {
-    return 'scale(' + scale + ') translateX(-' + pos + '%)';
+function tagTransform(scale, x, y) {
+    return `scale(${scale}) translateX(-${x}%) translateY(${y}%)`;
```
Comment (popojargo) [inline static/script/beer_carousel.js]: On serait mieux de pas utiliser les template string pour avoir un maximum de compatibilité

Comment (wilomgfx) [reply static/script/beer_carousel.js]: Unfortunately oui haha pas supporté par ie 11 :( https://caniuse.com/#feat=template-literals

Comment (Bazooo) [reply static/script/beer_carousel.js]: fixed

**Comment by Bazooo** [comment]
Added drag and fix animations not working

**Pull request merged**